docs/DEBUGGING.md

```diff
@@ -83,192 +83,192 @@ If BootReason indicates the system crashed due to a touch file, this means that
 
 The output will be a single long line with carriage returns and tabs as escaped characters. Fix it by replacing the characters and dumping them to a file:
 
-```sh
+```text
```

Comment:
I guess this didn't get rid of the complaints? 
It isn't important but a bit annoying that there isn't a way to get markdownlint to accept verbatim input.
But I'll let it slide.

Reply:
Yes @eriknordmark, I changed to "text" as you suggested in the original PR. Regarding hard tabs, from markdownlint documentation for the Rule MD010 (no hard tabs - https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md):

`You have the option to exclude this rule for code blocks and spans. To do so, set the code_blocks parameter to false. Code blocks and spans are included by default since handling of tabs by Markdown tools can be inconsistent (e.g., using 4 vs. 8 spaces).`

Well, I would still insist to keep the spaces in our documentation files in order to avoid an eventual mishandling of these tabs for any markdown processing tool. Also, for code blocks of other languages we must want to have this checking in order to avoid mixes of hard tabs vs spaces (this is quite annoying and usually common to happen).

In summary, if you agree, let's keep this PR as it is...